myokit/tests/test_formats_importers.py

```diff
@@ -82,6 +88,145 @@ def sbml(fname):
         # Model with listOfInitialValues and unit with multiplier
         sbml('Noble1962-initial-assignments-and-weird-unit.xml')
 
+    def test_parameters(self):
+        # Test Case: Hodkin Huxley
+        # expected
+        parameters = [
+            'V',
+            'V_neg',
+            'E',
+            'I',
+            'i_Na',
+            'i_K',
+            'i_L',
+            'm',
+            'h',
+            'n',
+            'E_R',
+            'Cm',
+            'g_Na',
+            'g_K',
+            'g_L',
+            'E_Na',
+            'E_K',
+            'E_L',
+            'V_Na',
+            'V_K',
+            'V_L',
+            'alpha_m',
+            'beta_m',
+            'alpha_h',
+            'beta_h',
+            'alpha_n',
+            'beta_n'
+        ]
+
+        # test whether parameters are in myokit model
+        for param in parameters:
+            assert self.HoHu.has_variable('sbml.' + param)
```

Comment:
I guess you've been using `pytest` :D 

In Myokit we're using plain `unittest` https://docs.python.org/3/library/unittest.html so this becomes `self.assertTrue(self.model.has_variable(..))`

Reply:
Ok, changed.